src/GraphQL/loaders/usersMutations.js

```diff
@@ -24,19 +18,30 @@ const load = parseGraphQLSchema => {
         type: parseGraphQLSchema.parseClassTypes['_User'].signUpInputType,
       },
     },
-    type: new GraphQLNonNull(defaultGraphQLTypes.SIGN_UP_RESULT),
+    type: new GraphQLNonNull(parseGraphQLSchema.meType),
     async resolve(_source, args, context) {
       try {
         const { fields } = args;
         const { config, auth, info } = context;
 
-        return await objectsMutations.createObject(
+        await objectsMutations.createObject(
           '_User',
           fields,
           config,
           auth,
           info
         );
+
+        return (await usersRouter.handleLogIn({
```

Comment:
It will turn out creating two session tokens. Can you try to use the same approach used by the `me` query [here](https://github.com/parse-community/parse-server/blob/master/src/GraphQL/loaders/usersQueries.js#L17)?